analysis/preprocess/Stage1_data_cleaning.R

```diff
@@ -194,7 +194,8 @@ consort[nrow(consort)+1,] <- c("Quality assurance: Pregnancy/birth codes for men
 print('Quality assurance: HRT or COCP meds for men')
 
 #input <- input[!(input$cov_cat_sex=="Male" & input$qa_bin_hrtcocp==TRUE),]
-input <- input[!(input$cov_cat_sex == "Male" & (input$cov_bin_combined_oral_contraceptive_pill | input$cov_bin_hormone_replacement_therapy) == TRUE),]
+# input <- input[!(input$cov_cat_sex == "Male" & (input$cov_bin_combined_oral_contraceptive_pill | input$cov_bin_hormone_replacement_therapy) == TRUE),]
+input <- input[!(input$cov_cat_sex == "Male" & (as.logical(input$cov_bin_combined_oral_contraceptive_pill) | as.logical(input$cov_bin_hormone_replacement_therapy))), ]
```

Comment:
I think this is fine but the following would also work:
`input[!(input$cov_cat_sex == "Male" & (input$cov_bin_combined_oral_contraceptive_pill==TRUE | input$cov_bin_hormone_replacement_therapy==TRUE)), ]`